bin/aliases.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+alias deploy-stage-edxapp="ansible-playbook edxapp.yml -e 'env=stage machine=app1' "
```

Comment:
Hrm, this may be the best way to do this for now, instead of of dozens of 2 line files..